### ISSUE TITLE
Finish Generating Wiktionary Offline App (English)

### ISSUE THREAD

**Issue by RavanJAltaie** [kiwix/kiwix-android-custom#74]
Could you please help in finishing generating Wiktionary Offline App?
The play store entry has been uploaded.
Please create the app itself (package Kiwix+zim file).
https://play.google.com/console/u/1/developers/9116215767541857492/app/4972878495171866290/main-store-listing
The Zim file is:
https://download.kiwix.org/zim/wiktionary/wiktionary_en_all_maxi_2023-02.zim

**Comment by gouri-panda** [comment]
@RavanJAltaie Sure!

**Comment by Popolechien** [comment]
@gouri-panda The zim is 6.6 GB even as nopic, we'll need to find / generate a smaller version first (see [ zim-requests 578](../../../openzim/zim-requests/issues/578))

**Comment by gouri-panda** [comment]
@Popolechien Ok. I'll wait until it comes out.

**Comment by kelson42** [comment]
@gouri-panda what is the status here. Can we have it in internal testing?

**Comment by gouri-panda** [comment]
@kelson42 Ok

**Comment by kelson42** [comment]
@Popolechien As such I guess this app will never go live because:
* we don't have an app specific ZIM files (could be fixed quickly)
* the welcomepage is not mobile friendly

**Comment by Popolechien** [comment]
@kelson42 Yes we need to be able to generate a selection akin to what we had discussed and achieved in openzim/mwoffliner/issues/989
The landing page issue is a trivial fix inasmuch as we can generate a quick, custom one as we had for the larger Wikipedias

How should we proceed?
